pensjon-brevbaker/src/test/kotlin/no/nav/pensjon/brev/IntegrationTestUtils.kt

```diff
@@ -40,14 +46,61 @@ fun requestLetter(client: HttpClient, letterRequest: BestillBrevRequest<Brevkode
     }
 
 fun writeTestPDF(pdfFileName: String, pdf: ByteArray, path: Path = Path.of("build", "test_pdf")) {
-    val file = path.resolve("$pdfFileName.pdf").toFile()
+    val file = path.resolve("${pdfFileName.replace(" ", "_")}.pdf").toFile()
     file.parentFile.mkdirs()
     file.writeBytes(pdf)
     println("Test-file written to file:${"\\".repeat(3)}${file.absolutePath}".replace('\\', '/'))
 }
 
 private val laTeXCompilerService = LaTeXCompilerService(PDF_BUILDER_URL, maxRetries = 0)
 
+fun renderTestPdfOutline(
```

Comment:
Kanskje se på om disse to bør forenes med eller benytte seg av `createVedleggTestTemplate` og `outlineTestTemplate`

Reply:
Fellesnevner ble selve create template og attachment funksjonen. Flyttet alt til samme fil iallefall.